src/redux/items/thunks.js

```diff
@@ -1,8 +1,10 @@
 import { createAsyncThunk } from '@reduxjs/toolkit'
 import axios from 'axios'
+const axios_ = require('axios')
```

Comment:
why is axios being imported twice?

Reply:
Just for testing purposes, I will delete it